src/main/kotlin/org/malachite/estella/people/domain/JobSeekerRepository.kt

```diff
@@ -6,4 +6,6 @@ import java.util.*
 interface JobSeekerRepository {
     fun findByUserId(user_id: Int): Optional<JobSeeker>
     fun save(updatedJobSeeker: JobSeeker): JobSeeker
+    fun findAll(): MutableIterable<JobSeeker>
+    fun deleteById(id: Int): Optional<JobSeeker>
```

Comment:
W offer mamy bez i pytanie czy mamy wobec tego konwencję czy też olewamy?

Reply:
Wydaje mi się, że to też trzeba obgadać, albo przynajmniej zobaczyć, jak standardowo hibernate robi XD

Reply:
do usunięcia typ zwracany w DELETE